src/components/Users/User.js

```diff
@@ -1,22 +1,27 @@
 import * as React from "react";
-import { Link } from "react-router-dom";
```

Comment:
unused...the rest of the file is prettier, except the addition of the alt on the img tag